pkgs/os-specific/linux/r8168/default.nix

```diff
@@ -53,7 +53,7 @@ in stdenv.mkDerivation rec {
     '';
     license = licenses.gpl2Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ timokau ];
-    broken = (lib.versions.majorMinor kernel.modDirVersion) != "5.15";
+    maintainers = with maintainers; [];
+    broken = (lib.versions.majorMinor kernel.modDirVersion) > "6.8";
```

Comment:
```suggestion
    broken = lib.versionAtLeast kernel.modDirVersion "6.9";
```

Reply:
can do - saw a reference to this syntax on an old PR but couldn't find a modern example - wasn't sure if the syntax was still valid